data/tlg2032/tlg001/__cts__.xml

```diff
@@ -0,0 +1,8 @@
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg2032" xml:lang="grc" urn="urn:cts:greekLit:tlg2032.tlg001">
+    <ti:title xml:lang="lat">Synagoge</ti:title>
+  <ti:edition urn="urn:cts:greekLit:tlg2032.tlg001.1st1K-grc1" workUrn="urn:cts:greekLit:tlg2032.tlg001">
+      <ti:label xml:lang="lat">Synagoge</ti:label>
+      <ti:description xml:lang="mul">Pappus Alexandrinus, Synagoge, Hultsch, Weidmann, 1876</ti:description>
```

Comment:
I would actually add this information to the text of the `<ti:description>` element so that it shows up on the GH page and people can see immediately that it is not the full work. I would simply write "Pappus Alexandrinus, Synagoge (Book 1), Hultsch, Weidmann, 1876"